### PR TITLE
Minor bug fix: Fix error message for `retries`

### PR DESCRIPTION
**Pull request by AD7six**
### Summary

Fix this error message which [refers to the wrong variable](https://github.com/Kong/kong/blob/c498f41dc03f7a04aaf3df24c59e01ff5e39c339/kong/pdk/service.lua#L150):

```
if retries < 0 or retries > 32767 then
      error("port must be an integer between 0 and 32767: given " .. retries, 2)
```

### Checklist

- [ ] The Pull Request has tests
- [X] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/blob/main/README.md)
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - (No user facing impact, no PR created)

### Issue reference

This is an unreported bug.

### CODE REVIEW

**Comment by liverpool8056** [comment]
@AD7six good catch. Please add a changelog under changelog/unrelease/kong

**Comment by AD7six** [comment]
Sure thing. Here's another minor bug I noticed along the way :) https://github.com/Kong/kong/pull/13658

File: changelog/unreleased/kong/fix-retries-error-message.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+message: "Fix error message for invalid retries variable"
+type: bugfix
+scope: Core
```
Comment (liverpool8056) [inline changelog/unreleased/kong/fix-retries-error-message.yml:3]:
```suggestion
scope: PDK
```

**Review by liverpool8056 — CHANGES_REQUESTED** [review]

**Review by liverpool8056 — APPROVED** [review]

**Comment by AD7six** [comment]
@liverpool8056 👋 what is required to get this MR merged :) ?

**Review by gszr — APPROVED** [review]
Nice catch!

**Comment by gszr** [comment]
Thanks for your first contribution, @AD7six! Here's to many more 🦍

**Comment by gszr** [comment]
> @liverpool8056 👋 what is required to get this MR merged :) ?
> 
> If it's waiting for some action from me - what is that action?

All PRs require two reviews : )

**Pull request merged**